.scripts/Generate-Api.ps1

```diff
@@ -0,0 +1,41 @@
+$repoRoot = Join-Path $PSScriptRoot .. -Resolve
+$dotnetFolder = Join-Path $repoRoot .dotnet\src
+
+$assemblyPath = Join-Path $dotnetFolder bin\Debug\netstandard2.0\OpenAI.dll
+$apiPath = Join-Path $repoRoot .dotnet\api\api.md
```

Comment:
Let's call the file `OpenAI.netstandard2.0.cs`.

Reply:
done -- retaining the .md file in parallel for diff but will remove when we're comfortable merging.